src/gamepad.cpp

```diff
@@ -257,7 +263,7 @@ void Gamepad::process()
 	}
 
 	// 4-way before SOCD, might have better history without losing any coherent functionality
-	if (options.fourWayMode) {
+	if (fourWayMode) {
```

Comment:
And because we're doing a toggle, we can just do options.fourWayMode ^ map48WayModeSwitchOn or something similar